src/alire/alire.ads

```diff
@@ -9,7 +9,7 @@ with Simple_Logging;
 
 package Alire with Preelaborate is
 
-   Version : constant String := "1.1.0-dev";
+   Version : constant String := "1.1.0-dev+c122dc5d";
```

Comment:
Why change the version here?

Reply:
This is to have an easy unique id in case someone reports problems using a build from master; otherwise all share the same version.